### PR TITLE
Add form validation using JavaScript and Constraint Validation API

### PR DESCRIPTION
**Pull request by gnuion**
Here We:
- Add form validation for email input, so it is all lowercase, using JavaScript.
- Use the HTML5 Constraint Validation API together with JavaScript

### CODE REVIEW

File: validate.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+});
+
+email.addEventListener('input', () => {
+  if (email.value !== email.value.toLocaleLowerCase()) {
+    email.setCustomValidity('Email should be all lowercase!');
+    email.reportValidity();
+  } else {
+    email.setCustomValidity('');
+  }
+});
```
Comment (tresorsawasawa) [inline validate.js:23]: - [ ] Kindly make sure the error message shows up close to the **"Get In Touch"** button as required for the project. I suggest you display it between the **Textarea** field and the **Get In Touch** button (i.e.: above the **Get In Touch"** button)

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @gnuion 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: validate.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const form = document.querySelector('form');
+
+const { email, button } = form.elements;
+
+const errSpan = document.createElement('p');
+errSpan.style.textAlign = 'left';
+errSpan.style.color = 'red';
+errSpan.style.fontSize = '1rem';
+
+button.insertAdjacentElement('beforebegin', errSpan);
```
Comment (ShahierNashaat) [inline validate.js:10]: - [ ] Kindly be noticed that this line is causing an error and it stops the validation, please fix that.

Screenshot from 2022-07-21 16-59-16

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 

Hi @gnuion, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] Good commit history :heavy_check_mark: 
- [ ] No linters errors :100: 
- [ ] Good implementation of all changes required :heavy_check_mark: 
- [ ] Good design :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**